perfkitbenchmarker/managed_relational_db.py

```diff
@@ -0,0 +1,107 @@
+from abc import ABCMeta, abstractmethod
+import uuid
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+
+# TODO (ferneyhough): change to enum
+flags.DEFINE_string('database', None,
+                    'Managed database flavor to use (mysql, postgres)')
+flags.DEFINE_string('database_name', None,
```

Comment:
Is database name dependent on the test tool? e.g., sysbench requires the database to be named as sbtest, I believe. (correct me if I am wrong here).